pizzabot.js

```diff
@@ -0,0 +1,119 @@
+//Iteration 1 - Setting everything up.
+const vegetarian = "vegetarian"
+const hawaiian = "hawaiian"
+const pepperoni = "pepperoni"
+const calzone = "calzone"
+const kebab = "kebab"
+
+// Iteration 4 - Finalize the order
+const calculateTotalCost = (event) => {
+    event.preventDefault();
+    //Get nr of pizzas
+    let orderQuantity = document.getElementById("nrOfPizzas").value;
+    //Get pizza sort
+    let orderName = document.getElementById("pizzaSort").value;
+    let pizzaPrice = pricePerPizza(orderName);
+    //Calculate totalcost for pizza
+    const orderTotal = orderQuantity * pizzaPrice
+    //Print out message to customer
+    let orderTime = calculateCookingTime(orderQuantity)
+    let pizzaImg = pizzaImages(orderName)
+    document.getElementById("pizzaImg").src = pizzaImg;
+    let info = `Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr. The pizza(s) will take ${orderTime} minutes.`;
+    //Set pizza picture to display block
+    let x = document.getElementById("pizzaImg");
+    x.style.display = "block";
+    //Display customer order confirmation
+    document.getElementById("submitInfo").innerHTML = info;
+
+};
+
+const form = document.getElementById("form");
+form.onsubmit = calculateTotalCost;
+
+//Sets the pizza image 
+const pizzaImages = (orderName) => {
+    let img;
```

Comment:
Good usage of `let`
Although, you could directly return the value and skip the variable.
```javascript
    if (orderName == `${vegetarian}`) {
        return "./Vegetariana.jpg";
    } else if (orderName == `${hawaiian}`) {
        return "./Hawaiian.jpg";
    } else if (orderName == `${pepperoni}`) {
        return "./Pepperoni.jpg";
    } else if (orderName == `${calzone}`) {
        return "./Calzone.jpg";
    } else if (orderName == `${kebab}`) {
        return "./Kebab.jpg";
    }
```

Reply:
In addition, you don't need to string template these variables, since the value is a string. 
```javascript
if (orderName == vegetarian) {
```
would suffice 👍

Reply:
Fixed! Removed string template variables and direct returns.